### PR TITLE
LPS-196539 Enforce variable name for PortletPreferences 1

### PR DESCRIPTION
**Pull request by qz1010**

### CODE REVIEW

**Comment by qz1010** [comment]
ci:test:sf

**Comment by qz1010** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java [hunk 1]
```diff
@@ -465,19 +465,19 @@ public List<Portlet> getEmbeddedPortlets() {
 
 	@Override
 	public List<Portlet> getEmbeddedPortlets(long groupId) {
-		List<PortletPreferences> portletPreferences = _getPortletPreferences(
-			groupId);
+		List<PortletPreferences> portletPreferencesList =
```
Comment (ling-alan-huang) [inline portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java:468]: List?

**Comment by ling-alan-huang** [comment]
@SylviaLuan Can I forward:force this PR? Thanks.
/cc @austinchiang

**Comment by SylviaLuan** [comment]
ci:forward:force

**Pull request closed without merging**